src/ui_framework/components/button/_button.scss

```diff
@@ -80,23 +80,23 @@
   }
 }
 
+/**
```

Comment:
That's fine, but you moved this comment up from 106, so maybe it should be duplicated for line 76 and 106?